README.md

```diff
@@ -227,6 +227,55 @@ ginkgolinter checks the following:
 * If the first parameter is a function with the format of `func(error)bool`, ginkgolinter makes sure that the second 
   parameter exists and its type is string.
 
+### Wrong Usage of the `Succeed` gomega Matcher [BUG]
+The `Succeed` gomega matcher asserts that a *single* error value is nil. If this matcher is used against more than one
+value, it will always fail.
+
+The linter will not suggest a fix for this rule. This rule cannot be suppressed.
+
+These are legal forms of this matcher for sync assertions:
```

Comment:
not sure I would use the word "legal" here. WDYT about "valid"?

Reply:
done